src/lib/apiTypesV2/Subscription.cpp

```diff
@@ -97,6 +97,11 @@ std::string Subscription::toJson(void)
     jh.addNumber("throttling", this->throttling);
   }
 
+  if (this->notification.counter > this->notification.maxLimit)
+  {
+    jh.addString("status", "inactive");
+  }
+
```

Comment:
I thinks this should be removed.

toJson() is just for rendering the inforamtion you have in the DB. The subscription would already be in inactive state if the logic that I mention at https://github.com/telefonicaid/fiware-orion/pull/3890/files#r667810559 is implemented correctly.

Reply:
Fixed in https://github.com/telefonicaid/fiware-orion/pull/3890/commits/49b0c6df2c4d993b4e0fec160026fe0ac0017cb7